nixos/infrastructure/flyingcircus-physical.nix

```
@@ -159,7 +159,7 @@ mkIf (cfg.infrastructureModule == "flyingcircus-physical") (lib.mkMerge [
         notification = "Network interfaces are healthy";
         command = "sudo ${fc.sensuplugins}/bin/check_interfaces " + (
            lib.concatMapStringsSep " "
-             (link: "-i ${link.link},1000:10000")
+             (link: "-i ${link.link},${toString (trace link link.minimumPortSpeed)}:")
```

```suggestion
             (link: "-i ${link.link},${toString link.minimumPortSpeed}:")
```

leftover debugging trace

---

Also no security implications described.